### PR TITLE
feat: add pngtodisk driver

### PR DESCRIPTION
**Pull request by cristianmtr**
Add a PngToDiskDriver, that stores intermediate representations of pngs to the workspace. 

Useful in debugging.

Think of it like `LogInfoDriver`, but for images.

- [ ] decide where helper methods should be stored
- [x] add unit test
- [x] add integration test
- [x] allow specifying workspace in a better way

Closes https://github.com/jina-ai/jina/issues/2112

### CODE REVIEW

File: jina/drivers/control.py [hunk 1]
```diff
@@ -158,3 +160,46 @@ def __call__(self, *args, **kwargs):
...
+    ) -> None:
+        def _move_channel_axis(
+            img: 'np.ndarray', channel_axis_to_move: int, target_channel_axis: int = -1
+        ) -> 'np.ndarray':
+            if channel_axis_to_move == target_channel_axis:
+                return img
+            return np.moveaxis(img, channel_axis_to_move, target_channel_axis)
+
+        def _load_image(blob: 'np.ndarray', channel_axis: int):
+            from PIL import Image
```
Comment (JoanFM) [inline jina/drivers/control.py:195]: as an external import, it should be imported with the help of `ImportChecker`

**Review by JoanFM — COMMENTED** [review]
If we go for this, I would add these kind of drivers into a `debug.py` new file. Would not merge with actual functional ones.

File: jina/drivers/control.py (hunk 1 above)
Comment (cristianmtr) [reply jina/drivers/control.py]: I can't find any usages of this class

It's added here but this method is never called

def check(args: 'Namespace'):
    """Check jina config, settings, imports, network etc"""
    from jina.checker import ImportChecker

    ImportChecker(args)

File: tests/integration/debug/components.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    def __init__(self, channel_axis: int = -1, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.channel_axis = channel_axis
+
+    @single(slice_nargs=2)
+    def craft(self, buffer: bytes, uri: str, *args, **kwargs) -> Dict:
+        """
+        Read image file and craft it into image matrix.
+
+        Read the image from the given file path that specified in `buffer` and save the `ndarray` of the image in
```
Comment (theUnkownName) [inline tests/integration/debug/components.py:31]: Super small typo: "That **is** specified"

File: tests/integration/debug/components.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    The coordinate is the same coordinate-system in the
+        :py:mode:`PIL.Image`.
+
+    :param target_size: Desired output size. If size
+        is a sequence like (h, w), the output size will
+        be matched to this. If size is an int, the
+        output will have the same height and width as
+        the `target_size`.
+    :param channel_axis: Axis for channel
```
Comment (theUnkownName) [inline tests/integration/debug/components.py:65]: Maybe it's better to specify it's the color channel?

Comment (cristianmtr) [reply tests/integration/debug/components.py:65]: This is copy-pasted from the Hub and only used in the test. I don't think we should modify them here

File: tests/integration/debug/components.py (hunk 2 above)
Comment (cristianmtr) [reply tests/integration/debug/components.py:31]: Also copy-pasted from the hub just for this test

**Review by nan-wang — DISMISSED** [review]
LGTM👍

**Comment by cristianmtr** [comment]
> LGTM

Reverted some changes I made while debugging CI and some unnecessary changes in control drivers

**Review by hanxiao — APPROVED** [review]

**Pull request merged**